src/main/java/app/GameApplication.java

```diff
@@ -1,7 +1,7 @@
 package app;
 
 import data_access.DBSearchDataAccessObject;
-import interface_adapter.PlayGameController;
+import interface_adapter.PlayGameAspects.PlayGameController;
 import interface_adapter.search.SearchViewModel;
 import use_case.search.SearchInteractor;
```

Comment:
Updated to use PlayGameAspects. Ensure documentation is updated to explain this architectural choice.